### DISCUSSION TITLE
use longhorn node to supply volumes to a different node

### DISCUSSION THREAD

**Discussion by cyclingwithelephants** [longhorn/longhorn#2605] [Q&A]
Hi folks, 

I've got a two node cluster, essentially a "compute" node (node A) with only an OS drive, and a small NAS node (node B) with an SSD that I'm exposing to longhorn. I'm looking to move to a more appropriate setup but for now this is what I've got.

With this in mind, I'm hoping to run workloads on node A with the storage backed by node B, is this possible? If so, where might I find an example please?

I suppose an equivalent question is if I were to have a 3 node cluster (already running with longhorn on each node) and wanted to add a fourth node, would that new node also require storage in order to be able to run workloads that require persistent volumes or would it be able to take advantage of the existing storage in the original three nodes over the network? (And if so how?)

**Comment by cclhsu** [comment]
Yes, this can be achieved through taint and toleration.  Please refer to the guide https://longhorn.io/docs/1.1.1/advanced-resources/deploy/taint-toleration/ for setup.

**Reply by cyclingwithelephants** [reply]
I moved this question to https://github.com/longhorn/longhorn/issues/2614 where I gave more info
